<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBWalkingDialog</class>
 <widget class="QDialog" name="HBWalkingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref torque</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No Of Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_DATA_SAVE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>data save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WALK_TEST">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Position Walking</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_TIME">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_STRIDE">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_TORQUE_TEST">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Torque Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_TEST_START">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_REF_TORQUE">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WALK_READY">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Walk Ready</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NO_OF_STEP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Stride(m)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_TEST_STOP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_APPLY">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>530</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Time(s)</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BT_SYSID_INPUT_START">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_SYSID_INPUT_FREQ">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>COM System ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Freq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_SYSID_INPUT_AMP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Amp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_SYSID_STOP_SAVE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>76</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop &amp; save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_SYSID_INPUT_START_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step Input</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BT_WALK_TEST_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Impact !</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IMPACT_TIME">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IMPACT_FORCE">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>469</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.01</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>474</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>474</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Force</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_CONTROL_ON">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Control On</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BT_OLTORQUE_GO">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mA imput Go</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_OLTORQUE_MA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OL Torque Tuning</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_ZERO_GAIN">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ZERO gain</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BT_ONE_LEG_READY">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Stance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_INV_DYNAMICS_CON">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Inv Dynamics Con</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_DYN_STANDING">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dyn Standing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_POS">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos Standing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_DYN_WALKING">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dynamics Walking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_JUMP">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Jump</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_JUMP_READY">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Jump Ready</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_COM_HEIGHT">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get COM Height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_LOGGING">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_ONE_LEG_READY_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Stance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_READY_TO_WALK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready to Walk</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
